crates/wasmtime/src/config.rs

```diff
@@ -391,6 +392,20 @@ impl Config {
     /// Creates a new configuration object with the default configuration
     /// specified.
     pub fn new() -> Self {
+        Self::new_with_isa_flags(native::builder())
+    }
+
+    /// Creates a [`Config`] for the given target triple.
+    ///
+    /// No CPU flags will be enabled for the config.
+    pub fn for_target(target: &str) -> Result<Self> {
```

Comment:
Instead of being a separate constructor could this be a configuration method?

We could document `Config::new` as matching whatever the native machine is and then a `.target(..)` method which resets that internally?

Reply:
That sounds good 👍  I'll make that change.